### ISSUE TITLE
[Issue] Script search in options goes very slow from v4.5 to v4.8

### ISSUE THREAD

**Issue by aompqn** [Tampermonkey/tampermonkey#691]
(Please fill out the issue template with your details)

## Expected Behavior
Please add old version search in options

## Actual Behavior
Script searching in options goes very slow from v4.5 to v4.8. From immediately to 10seconds

## Specifications

- Chromium: supported
- TM: 4.8
- OS: windows

## Script
Possible it is here 4.8_0\layout\default\options.js
(Please give an example of the script if applicable.)

```js
// ==UserScript==
// @name MyName
// @namespace http://mywebsite.com
// @Version 1.0
// @description try to take over the world!
// @author Me
// @match http://mywebsite.com/*
// @require http://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js
// @include https://mywebsite.com/*
// @grant none
// ==/UserScript==
```

**Comment by derjanb** [comment]
This is because the script content now is searched as well.

**Comment by derjanb** [comment]
Fixed at Tampermonkey 4.10.
